_data/ru/stage3.yml

```
@@ -0,0 +1,253 @@
+  authors:
+    - Daniel Ehrenberg
+    - Kevin Gibbons
+  champions:
+    - Daniel Ehrenberg
+    - Jeff Morrison
+    - Kevin Smith
+    - Kevin Gibbons
+  tests:
+    - 'https://github.com/tc39/test262/pulls?q=is%3Apr+is%3Aclosed+private+fields'
+- title: Статические методы класса fields и private статические методы
+  id: proposal-static-class-features
+  example: |-
+    class ColorFinder {
+      static #red = &#x22;#ff0000&#x22;;
+      static #green = &#x22;#00ff00&#x22;;
+      static #blue = &#x22;#0000ff&#x22;;
+
+      static colorName(name) {
+        switch (name) {
+          case &#x22;red&#x22;: return ColorFinder.#red;
+          case &#x22;blue&#x22;: return ColorFinder.#blue;
+          case &#x22;green&#x22;: return ColorFinder.#green;
+          default: throw new RangeError(&#x22;unknown color&#x22;);
+        }
+      }
+
+      // Somehow use colorName
+    }
+  presented:
+    - date: January&#xA0;2019
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2019-01/jan-30.md#private-fields-and-methods-refresher
+  has_specification: true
+  description: Предложение по добавлению трех функций к классам JavaScript, основываясь на предложениях методов fields и private предыдущего класса.
+  authors:
+    - Daniel Ehrenberg
+    - Kevin Gibbons
+    - Jeff Morrison
+    - Kevin Smith
+  champions:
+    - Shu-Yu Guo
+    - Daniel Ehrenberg
+  tests:
+    - 'https://github.com/tc39/test262/pulls?q=is%3Apr+is%3Aclosed+static+fields'
+- title: грамматика Хашбанга
+  id: proposal-hashbang
+  example: |-
+    #!/usr/bin/env node
+    // in the Module Goal
+    export {};
+    console.log(1);
+  presented:
+    - date: November&#xA0;2018
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2018-11/nov-28.md#hash-bang-grammar
+  has_specification: true
+  description: Это предложение должно соответствовать де-факто использованию на некоторых хостах CLI JS, которые допускают Shebangs / Hashbang. Такие хосты удаляют хэшбанг для того, чтобы сгенерировать правильные исходные тексты JS перед тем, как перейти к JS-движкам в настоящее время. Это позволит унифицировать и стандартизировать то, как это делается.
+  authors:
+    - Bradley Farias
+  champions:
+    - Bradley Farias
+  tests:
+    - 'https://github.com/tc39/test262/pull/2065'
+- title: Топ-уровень <code>await</code>
+  id: proposal-top-level-await
+  example: |-
+    // file.html
+    &#x3C;script type=module src=&#x22;a.mjs&#x22;&#x3E;&#x3C;/script&#x3E;
+    // a.mjs
+    await import(&#x22;./b.mjs&#x22;);
+    // b.mjs
+    await import(&#x22;./a.mjs&#x22;);
+  presented:
+    - date: June&#xA0;2019
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2019-06/june-6.md#top-level-await-for-stage-3
+  has_specification: true
+  description: Top-level <code>await</code> позволяет нам полагаться на саму систему модулей для обработки всех этих обещаний, а также убедиться в том, что все хорошо скоординировано.
```

left untranslated until original wording is fixed

---

```suggestion
  description: Top-level <code>await</code> позволяет использовать модули как асинхронные функции. ECMAScript модули (ESM) могут ожидать внешние ресурсы, заставляя другие модули, которые их импортируют, ждать, прежде чем они начнут исполняться.
```
This is the last one